openstack/02a-MAClow/IEEE802154E.c

```diff
@@ -1127,11 +1140,17 @@ port_INLINE void activity_ti2(void) {
     }
 
     // add 2 CRC bytes only to the local copy as we end up here for each retransmission
-    packetfunctions_reserveFooterSize(&ieee154e_vars.localCopyForTransmission, 2);
+    packetfunctions_reserveFooterSize(&ieee154e_vars.localCopyForTransmission, LENGTH_CRC);
 
     // configure the radio for that frequency
     radio_setFrequency(ieee154e_vars.freq);
-
+    //check if the radio needs to use the default RF or a custom RF config and configure accordingly. 
+    if (ieee154e_vars.UseDefaultRFConfig)
+    {
+        radio_setConfig (ieee154e_vars.DefaultRFConfig);

```

Comment:
IEEE802154e.c file is a shared file for all OpenWSN port. 
If this function is called here, you need implement this function at each of the OpenWSN platforms.